pwiz_tools/Skyline/Properties/Settings.settings

```diff
@@ -761,5 +761,14 @@
     <Setting Name="DetectionsShowLegend" Type="System.Boolean" Scope="User">
       <Value Profile="(Default)">True</Value>
     </Setting>
+    <Setting Name="AreaCVRatioIndex" Type="System.String" Scope="User">
+      <Value Profile="(Default)">-1</Value>
+    </Setting>
+    <Setting Name="ShowIonMobility" Type="System.Boolean" Scope="User">

```

Comment:
Should the default be true simply because 1 important user requested it?

Reply:
I think any ion mobility user would find it valuable, and even if they don't want it right away they'll at least be pleased to know that it's available as they turn it off. Sue would also very much like to have this available BTW.

Reply:
Okay. Let's get it in before feature complete. You still need to write a test, though.